pkg/crds/yaml/generated/management.cattle.io_features.yaml

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
we don't need to generate this file since we are using `factory.BatchCreateCRDs` to register the CRDs